data-raw/chirps_monthly_afg_adm1_historical.R

```diff
@@ -0,0 +1,65 @@
+#' `chirps_monthly_afg_adm1_historical.csv`
+#' `Date:` 2024-04-22
+#' This script generates the chirps_monthly_afg_adm1_historical.csv
+#' 
+#' **Description:**
+#' Using GEE temporally aggregate CHIRPS daily data to monthly.
+#' We then run zonal means  for each yr_mo combination at the admin-1 level 
+#' For all of Afghanistan.
+#' 
+#' **Tip:* 
+#' Script can be run in the background (on Mac with terminal call)
+#' `caffeinate -i -s Rscript data-raw/chirps_monthly_afg_adm1.R`
+
+library(rgee)
+library(tidyrgee)
+library(rhdx)
+library(tidyverse)
+ee_Initialize()
```

Comment:
I was having [this issue](https://github.com/r-spatial/rgee/issues/355) authenticating, so used [this workaround](https://github.com/r-spatial/rgee/issues/355#issuecomment-1884215354), just FYI

Reply:
Interesting! yeah it seems the `{rgee}` authentication procedures are in some state of flux at the moment. I personally am not having issues with it, but seems alot are. 

A couple questions
1. Are you using Rstudio? 
2. Also have you followed the `{rgee}` [install/authentication instructions](https://github.com/r-spatial/rgee)? I think it sets up a conda env w/ miniconda (also downloaded/set up gcloud?)

It's probably an annoying default set up if you are using pyenv for everything else. But so far has been working for me at least